articles/virtual-desktop/remote-app-streaming/licensing.md

```diff
@@ -56,12 +56,17 @@ Here's a summary of the two types of licenses for Azure Virtual Desktop you can
   - Paid in advance through a subscription
   - Same cost per user each month regardless of user behavior
   - Includes entitlements to some other Microsoft products and services
+  - May include Windows Server host licensing (depends on specific license)
 
 - Per-user access pricing:
   - Grants Azure Virtual Desktop access rights for external users only
   - Pay-as-you-go through an Azure meter
   - Cost per user each month depends on user behavior
   - Only includes access rights to Azure Virtual Desktop
+  - Only supports Windows Client host licensing
```

Comment:
I think these two additions may confuse more than they clarify. Correct me if I'm wrong, but I don't think any of the M365 licenses include entitlement to Windows Server session hosts with AVD. If I'm reading the [licensing prerequisites table](https://docs.microsoft.com/en-us/azure/virtual-desktop/prerequisites#operating-systems-and-licenses) correctly, Windows Server session hosts always require an RDS CAL or RDS User Subscription License. I suggest removing these two additions; the callout below seems pretty explicit to me.

In future perhaps we should ensure that per-user access pricing is represented in our licensing prerequisites table, too.

Reply:
ok, that's valid.  It will be valid for the other PR as well then. ;)

Let me make the change.